### PR TITLE
Update api config and README file

### PR DESCRIPTION
**Pull request by ushergod**
1. add default value for ConfigParser
2. update the README file for how to run api

Signed-off-by: Sihan Wang <wsihan@vmware.com>

### CODE REVIEW

**Comment by ericwb** [comment]
Fixes Issue #52

File: README.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ from column import APIRunner
 api_runner = APIRunner()
 api_runner.run_module('localhost', remote_user=None)
 ```
+## Try api
```
Comment (ericwb) [inline README.md]: Replace with "To run the API"

File: column/__init__.py [hunk 2]
```diff
@@ -9,7 +10,7 @@
 
 import ConfigParser
 
-DEFAULT_CONF_FILE = '/etc/column/column.conf'
+DEFAULT_CONF_FILE = os.path.join('/etc/column', 'column.conf')
```
Comment (ericwb) [inline column/__init__.py]: So when running in a virtualenv like tox, using an absolute path can cause problems.

Comment (ericwb) [inline column/__init__.py]: Also you should have / at all in the strings.  Use:

os.path.join('etc', 'column', 'column.conf')

File: column/__init__.py [hunk 3]
```diff
@@ -18,5 +19,12 @@
 
 __version__ = '0.3.6'
 
-cfg = ConfigParser.ConfigParser(allow_no_value=True)
+defaults = {
+    'log_file': os.path.join('/var/log', 'column.log'),
```
Comment (ericwb) [inline column/__init__.py]: Also you should have / at all in the strings. Use:

os.path.join('var', 'log', 'column.log')

File: column/__init__.py [hunk 4]
```diff
@@ -18,5 +19,12 @@
...
+    'log_level': 'DEBUG',
+    'server': '127.0.0.1',
+    'port': '48620'
```
Comment (ericwb) [inline column/__init__.py]: shouldn't port be an int?

Comment (ushergod) [inline column/__init__.py]: it is tested, string is fine

File: column/__init__.py [hunk 5]
```diff
@@ -18,5 +19,12 @@
 
 __version__ = '0.3.6'
 
-cfg = ConfigParser.ConfigParser(allow_no_value=True)
+defaults = {
+    'log_file': os.path.join(os.sep, 'var', 'log', 'column.log'),
+    'log_level': 'DEBUG',
+    'server': '127.0.0.1',
+    'port': '48620'
```
Comment (ericwb) [inline column/__init__.py]: It may work as a string but the data type should be int

**Pull request merged**